src/styles/blocks/header.scss

```diff
@@ -34,9 +30,13 @@
 
   &__link {
     color: inherit;
-    text-decoration: none;
     text-transform: uppercase;
     font-size: 12px;
     font-weight: 500;
+    transition: color 0.3s linear 0s;
```

Comment:
`transition: color 0.3s linear;` 
i think there is no reason to specify 0s delay if it's 0)